jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/JdbcOutputFormat.java

```diff
@@ -18,19 +18,25 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.hive.ql.exec.FileSinkOperator.RecordWriter;
 import org.apache.hadoop.hive.ql.io.HiveOutputFormat;
+import org.apache.hadoop.hive.shims.ShimLoader;
 import org.apache.hadoop.io.MapWritable;
 import org.apache.hadoop.io.NullWritable;
 import org.apache.hadoop.io.Writable;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.hadoop.mapred.OutputFormat;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
 import org.apache.hadoop.util.Progressable;
+import org.apache.hive.storage.jdbc.dao.GenericJdbcDatabaseAccessor;
 
 import java.io.IOException;
 import java.util.Properties;
 
 public class JdbcOutputFormat implements OutputFormat<NullWritable, MapWritable>,
                                          HiveOutputFormat<NullWritable, MapWritable> {
 
+  private org.apache.hadoop.mapreduce.RecordWriter recordWriter;
```

Comment:
> These two objects can be moved to `public RecordWriter getHiveRecordWriter`, since they are not being accessed outside of the scope of that method.

Done.